<launch>
  <node name="robotiq_epick_simplecontroller_node" pkg="robotiq_epick_control" type="robotiq_epick_simplecontroller_node.py" output="screen"/>
  <node name="robotiq_epick_node" pkg="robotiq_epick_control" type="robotiq_epick_node.py" output="screen" />
</launch>